proposal/color-4-new-spaces.md

```diff
@@ -770,17 +769,37 @@ To serialize a non-legacy color `color`:
 
 ## Procedures
 
+### Normalizing a Color Space Name
+
+This procedure accepts a color space `name`, throws an error if the name is
+invalid, and returns a normalized version of the name when possible.
+
+* If `name` is not an unquoted string, throw an error.
+
+* If `name` is a [dashed ident], return `name`.
+
+  > Dashed ident's are valid custom color spaces, and case-sensitive.
+
+* Return the result of calling `string.to-lower-case(name)`.
+
+  > Non-[dashed ident]s represent known spaces, which are case-insensitive and
+  > serialized in their lower-case form. For future compatibility with new
+  > color spaces in CSS, we allow and normalize unknown names in this format.
+
+[dashed ident]: https://drafts.csswg.org/css-values-4/#typedef-dashed-ident
+[normalizing a color space name]: #normalizing-a-color-space-name
+
 ### Looking Up a Known Color Space
 
 This procedure accepts a `name`, and attempts to look up a [known color space]
 with a matching name. It throws an error if `name` is not a valid color space
 name, and either returns the known color space, or `null` if no color space is
 matched.
 
-* If `name` is not an unquoted string, throw an error.
+* Set `name` to the result of [normalizing a color space name] named `name`.
 
-* If `name` is the name of a [known color space], ignoring case, return the
-  matching [known color space].
+* If `name` is the name of a [known color space], return the matching [known
+  color space].
 
 * Otherwise, return `null`.
```

Comment:
Do we still need this procedure if we're _only_ dealing with known color spaces?

Reply:
I think so. It now handles several error cases, including the error if no known color space is found. We could split that out into each place it's used, but that seems like unnecessary duplication?